pkg/operator/ceph/file/filesystem.go

```diff
@@ -73,6 +73,15 @@ func createFilesystem(
 		return fmt.Errorf("failed to get filesystem %s: %+v", fs.Name, err)
 	}
 
+	// As of Nautilus, allow_standby_replay is a fs property so we need to apply it
+	if cephv1.VersionAtLeast(clusterInfo.CephVersionName, cephv1.Nautilus) {
+		if fs.Spec.MetadataServer.ActiveStandby {
+			if err = client.AllowStandbyReplay(context, fs.Namespace, fs.Name, fs.Spec.MetadataServer.ActiveStandby); err != nil {
+				return fmt.Errorf("failed to set allow_standby_replay to filesystem %s: %v", fs.Name, err)
+			}
+		}
+	}
+
 	// set the number of active mds instances
 	if fs.Spec.MetadataServer.ActiveCount > 1 {
```

Comment:
@batrick so you don't want to include that part in the at least Nautilus condition too?

Reply:
Oops I meant using a condition "if ! at least Nautilus".

Reply:
I'm not sure I understand.

I think you're wondering about: in Mimic/Nautilus, you still need to set `max_mds` but there is no need deactivate ranks.

Reply:
@batrick this issue is tracked here https://github.com/rook/rook/issues/2725 and I'm working on it.